### ISSUE TITLE
Examples don't work

### ISSUE THREAD

**Issue by NaBUru38** [sloopidoopi/Spout4Unity#6]
Hello, I've downloaded Spout4Unity but I can't make the examples stream the texture.

At the Sending example scene, I click on play and the RenderTexture gets updated when I move the cube. On the Spout receiver i open the "UnityRender" stream but I get a black image.

I've tried the four texture format values with no success. Am I doing anything wrong? Thanks!

**Comment by sloopidoopi** [comment]
Hard to say what the problem could be.
What Spout Receiver do you use? I tested it with the SpoutReceiver.exe that is in the Spout2/DEMO folder.
If you use another SpoutReceiver check if it has DX11 support.

**Comment by b0rt** [comment]
i do have the same problem, opening SimpleSender.unity, hitting 'play': 

**unity console:**

Spout sender creation with name UnitySender failed !
UnityEngine.Debug:LogWarning(Object)
Spout.Spout:CreateSender(String, Texture, Int32) (at Assets/Spout/Scripts/Spout.cs:352)
Spout.SpoutSender:_CreateSender() (at Assets/Spout/Scripts/SpoutSender.cs:160)
Spout.SpoutSender:_Update() (at Assets/Spout/Scripts/SpoutSender.cs:133)
Spout.SpoutSender:Update() (at Assets/Spout/Scripts/SpoutSender.cs:98)

There are problems with creating the sender UnitySender. Please check your settings or restart Unity.
UnityEngine.Debug:LogWarning(Object)
Spout.SpoutSender:_CreateSender() (at Assets/Spout/Scripts/SpoutSender.cs:164)
Spout.SpoutSender:_Update() (at Assets/Spout/Scripts/SpoutSender.cs:133)
Spout.SpoutSender:Update() (at Assets/Spout/Scripts/SpoutSender.cs:98)

**env**
- unity 5.0.1f3 on win.8.1 (all on and runing in x64)
- receiver: spout dx11 receiver and vvvv example (both are testet as working receivers)
- gfx: gtx 970

**try-and-fail**
-setting texture formats (all 4)
-enabling depth buffer on SpoutRenderTexture.0

thanks for any hints, and sorry if this is hijacking another problem

**Comment by sloopidoopi** [comment]
What setting do you have on your RenderTexture in Unity? Did you use the deault settings that comes with the demo? TextureFormat DXGI_FORMAT_R8G8B8A8_UNORM worked on my computer all the time.

**Comment by b0rt** [comment]
Yes, i tried that. I am going to try to compile NativeSpoutPlugin. I tried that yesterday, but that opened up a whole new can of worms (using vs2013 and latest spout sdk [2.004], i got several errors, of which i could solve some [like file dependencies] but not all [like some deprecated function warnings] ).. 
-  which version of spoutsdk did you compile the plugin with ?

**Comment by sloopidoopi** [comment]
The used Spout SDK version is V2.002.It was compiled with VS2010

**Comment by b0rt** [comment]
thx. i managed to compile it, but its the same binary/dll (?) thats being compiled, but i guess that was to be expected. i have to dig in deeper and do my homework, progressing further, i really want to use unity+spout. i'll let you know what i come up with, for now im fishing in the dark, maybee i can get some hints looking at the vvvv implementation or something..

strangely, on my collegues pc (using a gtx 660) its running out of the box (otherwise same system win8.1 x64)

**Comment by b0rt** [comment]
I did give it another try, unity5.3.3f1 and spout 2.004 ..demoscene works flawless with Spout DX11 receiver
